gateway.tf

```diff
@@ -219,8 +222,11 @@ resource "aws_api_gateway_integration_response" "api_settings_get_integration" {
   selection_pattern = aws_api_gateway_method_response.api_settings_get.status_code
   status_code       = aws_api_gateway_method_response.api_settings_get.status_code
   response_parameters = {
-    "method.response.header.Content-Length" = "integration.response.header.Content-Length",
-    "method.response.header.Content-Type"   = "integration.response.header.Content-Type"
+    "method.response.header.Content-Length"            = "integration.response.header.Content-Length",
+    "method.response.header.Content-Type"              = "integration.response.header.Content-Type",
+    "method.response.header.Cache-Control"             = "'no-store'",
+    "method.response.header.Pragma"                    = "'no-cache'",
+    "method.response.header.Strict-Transport-Security" = "'max-age=86400; includeSubDomains'"
```

Comment:
can we make the time value coming from a variable?

Reply:
done